src/connectors/beefy/v0/pools/updatePools.ts

```diff
@@ -0,0 +1,79 @@
+import { fetchTokens, fetchVaults } from "../analytics/external/beefy.api";
+import fs from "fs";
+import { Pool } from "../../../../utils/types/connector-types";
+import { Chain } from "../../../../utils/types/networks";
+const path = require("path");
+const _ = require("lodash");
+
```

Comment:
This function is really only for beefy. We don't use it to map to the new enum I introduced, we use it because beefy has other networks, we don't support.